examples/computer_vision/cifar10_tf_keras/cifar_model.py

```diff
@@ -12,9 +12,8 @@
 from tensorflow.keras.metrics import categorical_accuracy
 from tensorflow.keras.models import Sequential
 
-# This example is used in some examples with an explicit tf1 image,
+# This example is used in some tests with an explicit tf1 image,
 #  so we keep backwards compatibility.
```

Comment:
I'd drop this whole comment.

It's pretty clear from the code that it's a tf version compatibility thing.

Reply:
True, it's clear what the code does in what situations, but without the comment it's not clear why this example gets special treatment.